compiler/src/dotty/tools/io/Directory.scala

```diff
@@ -67,7 +67,7 @@ class Directory(jpath: JPath) extends Path(jpath) {
    *  until it runs out of contents.
    */
   def deepList(depth: Int = -1): Iterator[Path] =
-    if (depth < 0) list ++ (dirs flatMap (_ deepList (depth)))
+    if (depth < 0) list ++ (dirs flatMap (_ deepList depth))
```

Comment:
removed redundant parentheses

Reply:
This should be written with dots instead: `dirs.flatMap(_.deepList(depth))`.

Reply:
@odersky 
Thank you for your review!
I fixed this line as you mentioned. 🙏